embedded-graphics/src/primitives/polyline.rs

```diff
@@ -0,0 +1,189 @@
+//! The line primitive
+
+use crate::draw_target::DrawTarget;
+use crate::drawable::Drawable;
+use crate::drawable::Pixel;
+use crate::geometry::Dimensions;
+use crate::geometry::Size;
+use crate::pixelcolor::PixelColor;
+use crate::primitives::Primitive;
+use crate::style::PrimitiveStyle;
+use crate::style::Styled;
+use crate::{
+    geometry::Point,
+    primitives::line::{Line, LineIterator},
+};
+
+/// Polyline primitive
+///
+/// Creates an unfilled chained line shape
+///
+/// # Examples
+///
+/// ## Create some lines with different styles
+///
+/// ```rust
+/// use embedded_graphics::{
+///     pixelcolor::Rgb565, prelude::*, primitives::Line, style::PrimitiveStyle,
+/// };
+/// # use embedded_graphics::mock_display::MockDisplay;
+/// # let mut display = MockDisplay::default();
+///
+/// // TODO: Example
+/// # Ok::<(), core::convert::Infallible>(())
+/// ```
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Default)]
+pub struct Polyline<'a> {
+    points: &'a [Point],
+}
+
+impl<'a> Polyline<'a> {
+    /// Create a new polyline from a list of points or an iterator
+    pub fn new(points: &'a [Point]) -> Self {
+        Self { points }
+    }
+
+    /// Get the total number of points in this polyline
+    pub fn len(&self) -> usize {
```

Comment:
`len` could be misinterpreted as the length of the polyline in pixels. If `points` is a public field this function might not be necessary at all.